packages/mermaid/src/diagrams/flowchart/flowDb.js

```diff
@@ -149,6 +149,7 @@ export const addSingleLink = function (_start, _end, type) {
   if (linkTextObj !== undefined) {
     edge.text = sanitizeText(linkTextObj.text.trim());
 
+    edge.text = edge.text.replaceAll('\\"', '"');
```

Comment:
In case we turned off escapement, do we still need this line?

Reply:
Nope, forgot to remove it with the vertex one...